RSignatureVerifier/Bundle+Environment.swift

```diff
@@ -0,0 +1,40 @@
+extension Bundle: EnvironmentSetupProtocol {
+    var valueNotFound: String {
+        return "NONE"
```

Comment:
You can remove return if the swift version allows it.

Reply:
done